cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -60,8 +62,25 @@ func nodeRegister(csiDriverName, httpEndpoint string) {
 	}
 	klog.Infof("Registration Server started at: %s\n", socketPath)
 	grpcServer := grpc.NewServer()
+
 	// Registers kubelet plugin watcher api.
-	registerapi.RegisterRegistrationServer(grpcServer, registrar)
+	if runtime.GOOS == "windows" {
+		for true {
```

Comment:
Instead of retrying forever, do we want to return fatalf?

Reply:
try a few times(e.g. 10 times) is also an option, I am ok with that.